src/Microsoft.Sbom.Api/Workflows/SbomParserBasedValidationWorkflow.cs

```diff
@@ -293,4 +309,47 @@ private void ThrowOnInvalidInputFiles(List<FileValidationResult> fileValidationF
             throw new InvalidDataException($"Your manifest file is malformed. {invalidInputFiles.First().Path}");
         }
     }
+
+    private void AddInvalidComplianceStandardElementsToFailures(List<FileValidationResult> fileValidationFailures, HashSet<InvalidElementInfo> invalidElements)
+    {
+        if (invalidElements == null || !invalidElements.Any())
+        {
+            return;
+        }
+
+        switch (configuration.ComplianceStandard?.Value?.Name)
+        {
+            case "NTIA":
+                AddInvalidNTIAElementsToFailures(fileValidationFailures, invalidElements);
+                break;
+            case "None":
+                break;
+            default:
+                break;
+        }
+    }
+
+    private void AddInvalidNTIAElementsToFailures(List<FileValidationResult> fileValidationFailures, HashSet<InvalidElementInfo> invalidElements)
+    {
+        foreach (var invalidElementInfo in invalidElements)
```

Comment:
Suggestion: put a guard in this method, then just call the method from line 320. The guard would be something like:

```
if (configuration.ComplianceStandard?.Value?.Name != "NTIA")
{
  return;
}
```

Reply:
I'm hesitant to do this because I want to keep the switch case for future changes that might include other compliance standards. That way it is very easy to see all the places where you just need to another case for the new compliance standard. This switch case pattern is also utilized in other places in the code, so I'd rather keep it consistent by using it here too.